### ISSUE TITLE
Message incorrectly parsed when message contains Parser::HEADER_STOP_CR_LF

### ISSUE THREAD

**Issue by mattford** [stomp-php/stomp-php#93]
When a message contains \r\n\r\n, the header end detection will not work correctly, for example:

```
{
  "value1" : "hello world"
,
  "value2" : "2002-02-01"

}
```

The message body consists of "}", and the rest of the payload is interpreted as being part of the header

I resolved this by swapping the order of the checks for HEADER_STOP_LF and HEADER_STOP_CR_LF in Stomp\Transport\Parser::detectFrameHead, unfortunately I cannot submit a pull request as my company doesn't allow it.

**Comment by mattford** [comment]
A more elegant solution might be to look for the first occurrence of EITHER HEADER_STOP_LF or HEADER_STOP_CR_LF

**Comment by jmglsn** [comment]
I think we need to check if the right after the command part of the message a carriage return is set, in this case the header content marker must also make use of the carriage return - otherwise it's not expected.
I'll add a test case.

**Comment by jmglsn** [comment]
@mattford that's the correct way, checking for first for both of them. I created a test case for this.

Thanks for reporting this issue. I'll prepare fixes for our active versions.

**Comment by jmglsn** [comment]
Fixed in releases [4.3.0](https://github.com/stomp-php/stomp-php/releases/tag/4.3.0) and [3.0.7](https://github.com/stomp-php/stomp-php/releases/tag/3.0.7).

thx @mattford :+1:
